### ISSUE TITLE
Implement image pre-check method

### ISSUE THREAD

**Issue by maralski** [fastai/course-v3#82]
If there is a corrupt image in the train, validation or test set PIL.Image which is used by pytorch dataloader will raise an exception when it encounters the image. For example, if there is a zero byte image.

It may be worth implementing a pre-check method that checks validity of images before training.

Happy to implement it if you think it is a worthwhile idea.

**Comment by maralski** [comment]
Oops wrong repo
